playbooks/bats_pipeline_katello_nightly.yml

```diff
@@ -1,27 +0,0 @@
-- hosts: all
```

Comment:
Should we drop https://github.com/theforeman/forklift/blob/master/playbooks/bats_pipeline_foreman_nightly.yml at the same time for consistency?

Reply:
probably a good idea, yeah, even if unrelated to the original change here.